### ISSUE TITLE
Compile Error - Generating nanocat.1

### ISSUE THREAD

**Issue by ichilver** [nanomsg/nanomsg#803]
When compiling the nanomsg 1.0.0 I get the following error

```
[ 72%] Generating nanocat.1
Couldn't find a view in @views for document
  Use --trace for backtrace
make[2]: *** [nanocat.1] Error 1
make[1]: *** [CMakeFiles/man.dir/all] Error 2
make: *** [all] Error 2
```

This is how I created the error.  Running Ubuntu 14.04.5 LTS Trusty.

```
cd /tmp
tar xzvf nanomsg-1.0.0.tar.gz
cd nanomsg-1.0.0

mkdir build
cd build
cmake ..
cmake --build .
```

This is where it goes wrong.

I tried the following as well but same error

```
cd /tmp
tar xzvf nanomsg-1.0.0.tar.gz
cd nanomsg-1.0.0

./configure
make
```

Any ideas?  Could I be missing a dependency? and if so what?

Please note I have got asciidoctor installed.

As a work around I have edited the CMakeLists.txt file and changed line 93 from
`option (NN_ENABLE_DOC "Enable building documentation." ON)`
to
`option (NN_ENABLE_DOC "Enable building documentation." OFF)`

This now compiles ok.

Shouldn't NN_ENABLE_DOC be OFF by default and only turned ON if you specify --enable-docs on the command line

**Comment by crayfishuk** [comment]
It's dependent on asciidoctor being installed. If you uninstall it, it won't try to build the docs.

**Comment by gdamore** [comment]
we enable docs by default if you have asciidoctor. 

What version of asciidoctor do you have?   This looks like a bug there

**Comment by ichilver** [comment]
It's installed via apt-get from archive.ubunutu.com, trusty repository.

Running asciidoctor --version says its 0.1.4

**Comment by gdamore** [comment]
Seems that is an older version. Maybe a bug was fixed.  Current version is 1.5.4.  Can you update and try again?

**Comment by ichilver** [comment]
apt-get upgrade
apt-get upgrade-dist

Still running version 0.1.4 on Ubuntu 14.04

**Comment by gdamore** [comment]
its installable as a ruby gem.  can you check the asciidoctor site?

**Comment by gdamore** [comment]
Perhaps we should add a check to ensure that asciidoctor is "new" enough.  But to be honest I'm almost at the point where I just don't care about it -- its easily disabled for users that can't figure out how to get a new enough asciidoctor.  I'm just going to close it for now.

**Comment by jdowner** [comment]
Just wanted to leave some information here for anyone else who encounters this problem. I am on ubuntu 14.04 and nanomsg isn't a package that I can install directory from apt-get. So I have added it as a submodule of the project I am working on. To disable the documentation from running by default, I added the line,
```
set(NN_ENABLE_DOC Off CACHE INTERNAL "")
add_subdirectory(nanomsg)
```
to the top-level CMakeLists.txt.
